README.md

```diff
@@ -277,6 +277,69 @@ end
 #=> "error message"
 ```
 
+### [Block](https://hexdocs.pm/exceptional/Exceptional.Block.html)
+
+Kind of a combination of Elixir's normal
+[`with`](https://hexdocs.pm/elixir/Kernel.SpecialForms.html#with/1)
+special form in addition to a monad-style `do` pipeline.
+
+This automatically-wraps every return value with
+[`normalize`](https://hexdocs.pm/exceptional/Exceptional.Normalize.html).
+
+```elixir
+block do
+  a <- {:ok, 2}
+  b = a * 2
+  c <- {:ok, b * 2}
+  c * 2
+end
+#=> 16
+
+block do
```

Comment:
Is there a more descriptive term than `block`? `handleable`? `protected`? `errorable`? Block is very general and could mean anything when reading through the code.

Reply:
I was actually paralyzed via [Decision Paralysis ](https://xkcd.com/1801/) for a long while so I chose the most simple and descriptive name, even if it is overly generic, and left it up to you if you wanted it changed.  ^.^;

I did a search through a variety of libraries though and found it an unused function/macro name surprisingly, so it seemed worth grabbing.  :-)

I'd personally think `block` is overall fine though, it is a generic **block** that can be enhanced later with far more functionality if it is deemed useful, and renaming it later or having 20 slightly differently named variants for different purposes seems excessive to me.  ^.^;

I can rename it though if you wish?